sdk-exporter/prometheus/src/main/scala/org/typelevel/otel4s/sdk/exporter/prometheus/autoconfigure/PrometheusMetricExporterAutoConfigure.scala

```diff
@@ -45,6 +46,7 @@ import scala.util.chaining._
   * | otel.exporter.prometheus.without.type.suffix     | OTEL_EXPORTER_PROMETHEUS_WITHOUT_TYPE_SUFFIX     | If metrics are produced without a type suffix. Default is `false`.                       |
   * | otel.exporter.prometheus.without.scope.info      | OTEL_EXPORTER_PROMETHEUS_WITHOUT_SCOPE_INFO      | If metrics are produced without a scope info metric or scope labels. Default is `false`. |
   * | otel.exporter.prometheus.without.target.info     | OTEL_EXPORTER_PROMETHEUS_WITHOUT_TARGET_INFO     | If metrics are produced without a target info metric. Default is `false`.                |
+  * | otel.exporter.prometheus.shutdown.timeout        | OTEL_EXPORTER_PROMETHEUS_SHUTDOWN_TIMEOUT        | The time to wait for provider to do any cleanup required. Default is `10 seconds`.       |
```

Comment:
Could you please add a new configuration setting to these tables as well?

https://github.com/typelevel/otel4s/blob/main/docs/sdk/configuration.md?plain=1#L118

https://github.com/typelevel/otel4s/blob/ad1f20065a7a4666b103414752335400847eee23/sdk-exporter/prometheus/src/main/scala/org/typelevel/otel4s/sdk/exporter/prometheus/autoconfigure/PrometheusMetricExporterAutoConfigure.scala#L191

Reply:
fixed in https://github.com/typelevel/otel4s/pull/838/commits/ceb560b1cbbde4aa7f20aec31a276efda886e463